### DISCUSSION TITLE
this.model is null

### DISCUSSION THREAD

**Discussion by IcyYouk** [mozilla/hubs#6038] [Spoke Questions] [answered]
Hi !
I am trying to publish my HUBS-project but I have an error message : "this.model is null" (Error Publishing Project)
For a few days, I already tried to create the room (to see how it lokks like) and it was really good.
Can you please help me ? Thank you !
Best regards.
![Hubs](https://user-images.githubusercontent.com/130968015/232418710-7cc80396-d07b-4ca1-ac18-ff696e98cece.JPG)

**Comment by IcyYouk** [comment, marked as answer]
Problem was with a sketchlab asset, which disappeared.

**Reply by matthewbcool** [reply]
I can confirm deleting an object throwing this error solves the issue. Tested on the object below.
Thanks for following up @IcyYouk  

Of course, ideally we would like to handle errors like these more gracefully in Spoke with a more descriptive error message.


![f5f07c529a46c14d74e32f49d68fc3d0](https://user-images.githubusercontent.com/4493657/233758716-f01e318f-7ccc-4c82-a56f-e5c26ba0eeba.png)
